test/fixtures/functionMsk/core.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// NOTE: the `utils.js` file is bundled into the deployment package
+// eslint-disable-next-line
+const { log } = require('./utils');
+
+// NOTE: `kafkajs` is bundled into the deployment package
+// eslint-disable-next-line
+const { Kafka } = require('kafkajs');
+
+function consumer(event, context, callback) {
+  const functionName = 'consumer';
+  const { records } = event;
+  const messages = Object.values(records)[0].map(record =>
+    Buffer.from(record.value, 'base64').toString()
+  );
+  log(functionName, JSON.stringify(messages));
```

Comment:
We may just do `console.log`. As `log` from `utils`  is used in some tests purely to output _markers_ on which depends the deprecated [`waitForFunctionLogs`  util](https://github.com/serverless/serverless/blob/29d5fe75104c9bb0f44531e1c4b3491d4051f6b6/test/utils/integration.js#L71) (and we now use `confirmCloudWatchLogs`, which is more powerful)

Therefore I think we can remove `utils` file completely from this fixture